docs/versionhistory.rst

```diff
@@ -5,6 +5,8 @@ This library adheres to `Semantic Versioning 2.0 <http://semver.org/>`_.
 
 **UNRELEASED**
 
+- Fixed ``str`` and ``repr`` of ``MemoryObjectItemReceiver``, when ``item`` is not defined.
+  (`#767 <https://github.com/agronholm/anyio/pulls/767>`_)
```

Comment:
`str()` calls `__repr__()` if `__str__()` is not defined, so I feel mentioning `__repr__()` is sufficient.
```suggestion
- Fixed ``__repr__()`` of ``MemoryObjectItemReceiver``, when ``item`` is not defined
  (`#767 <https://github.com/agronholm/anyio/pulls/767>`_; PR by @Danipulok)
```